src/onnx/parse_split.cpp

```diff
@@ -55,12 +55,16 @@ struct parse_split : op_parser<parse_split>
         {
             literal s = parser.parse_value(info.attributes.at("split"));
             s.visit([&](auto v) { vec_splits.assign(v.begin(), v.end()); });
-
-            if(std::accumulate(vec_splits.begin(), vec_splits.end(), int64_t(0)) !=
-               static_cast<int64_t>(lens[tuned_axis]))
+        }
+        else if(args.size() == 2)
+        {
+            if(not args[1]->can_eval())
```

Comment:
Dont need for `can_eval` when calling `check_arg_empty`.

Reply:
done